### ISSUE TITLE
has the API to get the code with command ?

### ISSUE THREAD

**Issue by 2014qirong** [Sneezry/authenticator#124]
Does this have the API to get code with commands ?

**Comment by Sneezry** [comment]
No, Chrome extension does not have a CLI interface. But you can write a Node script with the totp library.

https://github.com/Sneezry/authenticator/blob/master/totp/totp.js
